internal/handlers/jsonb1/msg_count.go

```diff
@@ -45,7 +45,7 @@ func (s *storage) MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, e
 		"let",
 	}
 	if err := common.Unimplemented(document, unimplementedFields...); err != nil {
-		return nil, err
```

Comment:
No, `Unimplemented` returns protocol error that we should return as-is to the client